src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,22 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder buff = new StringBuilder();
+        String result = "0";
+        if (value > 0) {
+            while (value != 0) {
+                if (value % 2 == 0) {
+                    buff.append(0);
+                } else {
+                    buff.append(1);
+                }
+                value = value / 2;
+
+            }
+            buff.reverse();
+            result = buff.toString();
+            return result;
```

Comment:
buff.reverse();
result = buff.toString();
return result;
--> we can change 3 line to one -->
result = buff.reverse().toString;